uk_bin_collection/uk_bin_collection/councils/FarehamBoroughCouncil.py

```
@@ -35,43 +35,76 @@ def parse_data(self, page: str, **kwargs) -> dict:
+            params = {
+                "type": "JSON",
+                "list": dataset,
+                "Road or Postcode": user_postcode,
+            }
 
-        response = requests.get(
-            "https://www.fareham.gov.uk/internetlookups/search_data.aspx",
-            params=params,
-            headers=headers,
-        )
+            response = requests.get(
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)